system/Security/SecurityInterface.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+/**
+ * This file is part of the CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace CodeIgniter\Security;
+
+use CodeIgniter\HTTP\RequestInterface;
+use CodeIgniter\Security\Exceptions\SecurityException;
+
+/**
+ * Expected behavior of a Security.
+ */
+interface SecurityInterface
+{
+	/**
+	 * CSRF Verify
+	 *
+	 * @param RequestInterface $request
+	 *
+	 * @return $this|false
+	 * 
+	 * @throws SecurityException
+	 */
+	public function verify(RequestInterface $request);
+
+	//--------------------------------------------------------------------
```

Comment:
Please remove ALL code separator comments used in this file.

Reply:
Why? all methods in the framework interfaces are separated by comment line,

Reply:
_Not all_ interfaces, actually.

Up until now, code separator comments serve no real purpose than to delineate perfectly readable code. These just add unnecessary lines of non functional code.

Although these sort of comments are used frequently does not mean it is empirically right when it adds no value itself. We're just being economic here. Besides, there is an automated fix to remove all existing code separators if the need be, but it is just not the priority now.

Reply:
This is a style holdover from CI3 that we're slowly removing as we process files. While it provides a nice visual cue, most decent editors/IDEs today provide a way to do something similar. And if not, the comment block does a good job of it on its own.